localenv/mock-account-servicing-entity/Dockerfile

```diff
@@ -1,21 +1,59 @@
-FROM node:18.13.0-slim as builder
+FROM node:18-alpine AS base
 
-WORKDIR /workspace
+WORKDIR /home/rafiki
 
-RUN apt update
-RUN apt install -y curl xz-utils python3 build-essential
-
-# version in curl is not the version used. Dependent on the last command
 RUN corepack enable
 RUN corepack prepare pnpm@7.25.1 --activate
```

Comment:
Any particular reason we are still using `pnpm@v7` instead of `v8`?

Reply:
I guess we forgot to update it?

Reply:
Have you tried with pnpm v8?

Reply:
I have tried with pnpm v8 when I was running into the `httpbis-digest-headers` issue. It should also work with v8 without any problems. As for the choice to stick with v7, I recall @mkurapov mentioning an edge case or an issue that was preventing us from upgrading to v8, but I'm not entirely sure about this.